src/widgets/AddDashboardButton.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import AddIcon from '@material-ui/icons/AddCircle';
+import { colors } from '../modules/colors';
+
+const AddDashboardButton = () => {
+    return (
```

Comment:
Minor style issue but wouldn't this be cleaner without the functional body?  I.E. `const AddDashboardButton = () => (<React.Fragment>...</React.Fragment>)`

Reply:
Done